src/views/Joseki/Joseki.tsx

```diff
@@ -1444,6 +1447,10 @@ class EditPane extends React.Component<EditProps, any> {
         // Extract markup for "board marks"
         // maps markup of form "<label:position>"  to an array of {label, position} objects for each mark
 
+        if (description === null) { // I don't see how, but Sentry logs seem to imply there is a way!
```

Comment:
Magic!

Reply:
I'm not aware of a sentry report you filled out: I was responding to an email I got from Sentry, but I guess it could be the same?

Reply:
I used much more detail but essentially it was: 
go to Play tab, select a tag, select a contributor, change tag to the top option (empty) and the page crashes